### PR TITLE
Fixed login screen issues

### PR DESCRIPTION
**Pull request by mail2bishnoi**
Hi @tananaev 

I have fixed some of the issues on the login screen. Please review.

1. Login on pressing Enter key.
2. Replaced internal state-based navigation with the router.
3. Show/Hide Reset Password button.

Thanks.

### CODE REVIEW

File: modern/src/App.js [hunk 1]
```diff
@@ -43,7 +46,9 @@ const App = () => {
       <SocketController />
       <CachingController />
       <Switch>
-        <Route exact path='/login' component={LoginPage} />
+        <Route exact path='/login' component={LoginForm} />
+        <Route exact path='/register' component={RegisterForm} />
+        <Route exact path='/resetpassword' component={ResetPasswordForm} />
```
Comment (tananaev) [inline modern/src/App.js:51]: I would change the path to `/reset-password`.

File: modern/src/LoginPage.js [hunk 2]
```diff
@@ -42,12 +40,10 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const LoginPage = () => {
+const LoginPage = ({ children }) => {
```
Comment (tananaev) [inline modern/src/LoginPage.js:43]: It probably makes sense to rename `LoginPage` to something else. Maybe `StartPage` or `OnboardingPage` or something like that.

**Comment by mail2bishnoi** [comment]
Hi @tananaev 

Done.

**Comment by tananaev** [comment]
Merged, thank you.

**Comment by tananaev** [comment]
@mail2bishnoi it looks like this overlap issue is not resolved on the login:

Screen Shot 2021-06-26 at 4 44 57 PM

Can you please fix it if you have time.

**Comment by mail2bishnoi** [comment]
Hi @tananaev 

@dkyeremeh once told that the solution is in his mind and he will fix it.

**Comment by tananaev** [comment]
Yep, I already commented on the other thread.

**Pull request merged**